### PR TITLE
github/workflows: More build tests

### PR DESCRIPTION
**Pull request by PatrickRudolph**
Signed-off-by: Patrick Rudolph <patrick.rudolph@9elements.com>

### CODE REVIEW

**Comment by sylv-io** [comment]
create a PR for 9elements branch.

**Comment by PatrickRudolph** [comment]
I don't see why

File: .github/workflows/testing.yml [hunk 1]
```diff
@@ -4,8 +4,65 @@ name: Testing
...
+          git submodule update --recursive
+
+      - name: Build BaseTools
+        run: |
+          make -C BaseTools
+
+      - name: Build EmulatorPkg
+        run: |
+          source ./edksetup.sh
+          build -p EmulatorPkg/EmulatorPkg.dsc -t GCC5 -a X64
```
Comment (sylv-io) [inline .github/workflows/testing.yml:35]: shall we upload the artifacts of this plattform too?
So we could test individually without the need to build it ourself.

File: .github/workflows/testing.yml [hunk 2]
```diff
@@ -4,8 +4,65 @@ name: Testing
...
+
+      - name: Build BaseTools
+        run: |
+          make -C BaseTools
+
+      - name: Build IA32X64 OvmfPkg
+        run: |
+          source ./edksetup.sh
+          build -DSOURCE_DEBUG_ENABLE=TRUE -DSECURE_BOOT_ENABLE=TRUE -DTPM_ENABLE=TRUE -DTPM_CONFIG_ENABLE=TRUE -DNETWORK_HTTP_BOOT_ENABLE=TRUE -a IA32 -a X64 -t GCC5 -b DEBUG -p OvmfPkg/OvmfPkgIa32X64.dsc
+
```
Comment (sylv-io) [inline .github/workflows/testing.yml:65]: ^ same here

**Review by sylv-io — APPROVED** [review]

**Comment by sylv-io** [comment]
building artifacts could be added afterwards.

**Pull request merged**